packages/esm-patient-list-management-app/src/list-details-table/list-details-table.test.tsx

```diff
@@ -1,6 +1,29 @@
 import React from 'react';
-import { render, screen } from '@testing-library/react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
```

Comment:
```suggestion
import userEvent from '@testing-library/user-event';
import { render, screen, waitFor } from '@testing-library/react';
```

Prefer `userEvent` over `fireEvent`.